src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -168,6 +169,9 @@ const MigrateWalletCompletionComponent = (props: Props) => {
           let spendInfo: EdgeSpendInfo = {
             currencyCode: oldWallet.currencyInfo.currencyCode,
             spendTargets: [{ publicAddress: newPublicAddress }],
+            metadata: {
+              notes: sprintf(lstrings.migrate_wallet_tx_notes, getWalletName(newWallet))
+            },
```

Comment:
Can we do `category: "Transfer"` here? Also, Maybe put the wallet name also as `name: getWalletName(newWallet)`?